sunpy/util/parfive_helpers.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from functools import wraps
+
+import aiohttp
+import parfive
+from parfive import Results
+
+import sunpy
+
+__all__ = ['Downloader', 'Results']
+
+
+# Overload the parfive downloader class to set the User-Agent string
+class Downloader(parfive.Downloader):
```

Comment:
Why make a full class for this?
Wouldn't simply passing values be good enough?

Reply:
Because it makes it transparent to all the other code in the package that uses it, you just change the import and now it sets the header etc. I did consider defining `User-Agent` and then passing it `Downloader(headers=SUNPY_USER_AGENT)` everywhere but that felt like more effort.